hw1/index1.js

```diff
@@ -0,0 +1,6 @@
+const sayHello = function (name) {
+    return (name === 'Mark') ? 'Hi, Mark!' : ('Hello, ' + name + '!');
```

Comment:
What if I pass lower cased word 'mark'. Seems that code will not work as expected. Please, add logic to ignore case sensitive issue